server/form-pages/apply/about-the-person/personal-information/supportWorkerPreference.ts

```diff
@@ -37,7 +37,7 @@ export default class SupportWorkerPreference implements TaskListPage {
   }
 
   next() {
-    return ''
+    return 'gender'
```

Comment:
Should this have been kept as `''`? It's the last page in the journey

Reply:
oh whoops, this was left in from my initial implementation. thanks!